.github/scripts/redis_test/index.js

```diff
@@ -0,0 +1,232 @@
+  await failure(circuit2, { data: 'Circuit2' });
+  check(slidingTimeBreaker2.state === Mollitia.BreakerState.OPENED, 'Ok, circuit is opened', 'Circuit is not opened as expected...');
+  try {
+    await success(circuit1, { data: 'Circuit1' });
+    console.error('This request should not be a success as circuit is opened');
+    process.exit(1);
+  } catch (e) {
+    check(e.message === 'Circuit is opened', 'Ok, Request rejected due to circuit opened', 'Request should have been rejected with opened circuit');
+  }
+  await failure(circuit3, { data: 'Circuit3' });
+  await failure(circuit3, { data: 'Circuit3' });
+  check(slidingTimeBreaker2.state === Mollitia.BreakerState.OPENED, 'Ok, circuit is opened', 'Circuit is not opened as expected...');
+  check(slidingTimeBreaker3.state === Mollitia.BreakerState.CLOSED, 'Ok, circuit is closed', 'Circuit is not closed as expected...');
+  await failure(circuit3, { data: 'Circuit3' });
+  check(slidingTimeBreaker3.state === Mollitia.BreakerState.OPENED, 'Ok, circuit is opened', 'Circuit is not opened as expected...');
+}
+const testSlidingTimeBreakerModuleSlowRequest = async () => {
+  const breakerData = {
+      failureRateThreshold: 50,
+      openStateDelay: 10,
+      slidingWindowSize: 1000,
+      minimumNumberOfCalls: 2,
+      permittedNumberOfCallsInHalfOpenState: 1,
+      slowCallDurationThreshold: 100,
+      slowCallRateThreshold: 50,
+      redis: {
+        use: true
+      },
+      name: 'mySlidingTimeBreakerSlow'
+    };
+    const slidingTimeBreaker = new Mollitia.SlidingTimeBreaker(breakerData);
+    const slidingTimeBreaker2 = new Mollitia.SlidingTimeBreaker(breakerData);
+    const circuit1 = new Mollitia.Circuit({ options: { modules: [slidingTimeBreaker] } });
+    const circuit2 = new Mollitia.Circuit({ options: { modules: [slidingTimeBreaker2] } });
+    await success(circuit1, { data: 'Circuit1', delay: 150 });
+    await success(circuit2, { data: 'Circuit2' });
+    // Even if 50% of slow requests, circuit is kept closed as last request is success
+    check(slidingTimeBreaker2.state === Mollitia.BreakerState.CLOSED, 'Ok, circuit is closed', 'Circuit is not closed as expected...');
+    await success(circuit1, { data: 'Circuit1', delay: 150 });
+    check(slidingTimeBreaker.state === Mollitia.BreakerState.OPENED, 'Ok, circuit is opened', 'Circuit is not opened as expected...');
+    await delay(150);
+    check(slidingTimeBreaker.state === Mollitia.BreakerState.HALF_OPENED, 'Ok, circuit is Half Opened', 'Circuit is not Half Opened as expected...');
+    await success(circuit2, { data: 'Circuit2', delay: 150 });
+    check(slidingTimeBreaker2.state === Mollitia.BreakerState.OPENED, 'Ok, circuit is opened', 'Circuit is not opened as expected...');
+    await delay(10);
+    await expect(circuit.fn(successAsync).execute('dummy')).resolves.toEqual('dummy');
```

Comment:
Are those expects coming from `vitest`?
It doesn't look like it's a vitest test file

Reply:
Right, forgot to replace these ones.
Fixed